packages/ve-hemi-actions/test/actions/public/getTotalVeHemiSupplyAt.test.ts

```diff
@@ -0,0 +1,49 @@
+import { zeroAddress } from 'viem'
+import { readContract } from 'viem/actions'
+import { describe, expect, it, vi, beforeEach } from 'vitest'
+
+import { veHemiAbi } from '../../../abi'
+import { getTotalVeHemiSupplyAt } from '../../../actions'
+import * as constants from '../../../constants'
+
+vi.mock('viem/actions')
+
+describe('getTotalVeHemiSupplyAt', function () {
+  const mockClient = {
+    chain: { id: 1 },
+  }
+
+  const mockVeHemiAddress = zeroAddress
+  const mockTimestamp = 1761782400n
+  const mockTotalSupply = 11649344692938931129822560n
+
+  beforeEach(function () {
+    vi.clearAllMocks()
+  })
```

Comment:
We can remove this

```suggestion
```

By the [vitest config](https://github.com/hemilabs/ui-monorepo/blob/43d99b2ccf9642a89313746ff86f33033e2ba939/packages/ve-hemi-actions/vitest.config.ts#L5) mocks will always be cleared between test